test/test_files/strict_property_initialization/no_initialization.ts

```diff
@@ -0,0 +1,24 @@
+interface BasicInterface {}
+
+class StrictPropertyInitializationComponent {
+  // Test: Unassigned scalar property
+  // Fix: private unAssignedScalar?: boolean;
+  private unAssignedScalar: boolean;
+
+  // Test: Unassigned nonscalar property: list
+  // Fix: private unAssignedNonScalar?: [];
+  private unAssignedNonScalar: [];
+
+  // Test: Unassigned nonscalar property: interface
+  // Fix: private unAssignedInterface?: BasicInterface;
+  private unAssignedInterface: BasicInterface;
+
+  constructor() {
+    // Fix: this.takesInBoolean(this.unAssignedScalar!);
+    this.takesInBoolean(this.unAssignedScalar);
```

Comment:
if a variable is assigned, then it should not be added with "?", let's add an assignment to a variable here.